src/vfc_report

```diff
@@ -0,0 +1,545 @@
+#!/usr/bin/env python3
+import numpy as np
+import pandas as pd 
+import matplotlib.pyplot as plt
+import random
+import math
+import sys
+import struct
+import shlex
+import os.path
+import subprocess
+from matplotlib.patches import Patch
+from matplotlib.lines import Line2D
+from os import listdir
+
+DaraDir = "vfc_exp_data/"
+ImgDir = DaraDir + "img/"
+Arguments = DaraDir + "ArgumentsResults.csv"
+Operations = DaraDir + "OperationsResults.csv"
+ArgumentsAll = DaraDir + "AllArgsResults.csv"
+OperationsAll = DaraDir + "AllOpsResults.csv"
+Report_File = DaraDir + "vfc_precexp_report.html"
+
```

Comment:
I think this would have been much nicer with an HTML templating engine such as python jinja2 or some library for generating html programmatically. Given that you do not have much time, I'm ok merging this as it is for now. But in the future, I would like to rewrite this with a template engine.

Reply:
We can also remove it from this PR

Reply:
I think it is ok to keep it for now.